multi-anchor normative requirements in spec (#426)
HL7/fhircast-docs: input/pagecontent/4-5-multi-anchor-considerations.md

### Hub derives open events

> The hub is responsible for identifying and sending these implied open events. When distributing a received event, the Hub is responsible for generating and communicating open events for the resource types referenced by the received event. It's important that Hubs don't generate and send duplicative events. 

Reviewer: * Should the "open" in "communicating open events" be formatted as `open`?
* Last sentence, the spec generally avoids contractions - *could* expand these (fine either way, just calling attention).

> The hub is responsible for identifying and sending these implied open events. When distributing a received event, the Hub is responsible for generating and communicating open events for the resource types referenced by the received event. It's important that Hubs don't generate and send duplicative events. 

Author: ```suggestion
The hub is responsible for identifying and sending these implied open events. When distributing a received event, the Hub is responsible for generating and communicating open events for the resource types referenced by the received event. It is important that Hubs do not generate and send duplicative events. 

```

See details in the specification about:
* Hubs can [reject subscriptions](2-4-Subscribing.html#subscription-response) according to their own internal business logic.

> * Hubs are [required](2-5-EventNotification.html#hub-generated-open-events) to derive and send open events. 

Reviewer: Should the "open" in "send open events" be formatted as `open`?

Author: I'm gonna say no. See: https://build.fhir.org/ig/HL7/fhircast-docs/2-3-Events.html

We're only formatting specific events, or other terms that are placed in messages or code.  Any concerns, Gino?
